code/modules/clothing/suits/armor.dm

```diff
@@ -193,5 +193,4 @@
 /obj/item/clothing/suit/armor/tdome/green
 	name = "thunderdome suit"
 	desc = "Pukish armor."	//classy.
-	icon_state = "tdgreen"
-	item_state = "tdgreen"
```

Comment:
Is this intentional? If so, what does it do? Why is it not in the PR description?